pkgs/development/python-modules/pook/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+, furl
+, isPy3k
+, jsonschema
+, mock
+, nose
+, pytestCheckHook
+, requests
+, xmltodict
+}:
+
+buildPythonPackage rec {
+  pname = "pook";
+  version = "1.0.0";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "h2non";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1s2c0g4jis1i6sc2db1v6cr9faib6bcpjlvwnc866z45vsf70lvy";
+  };
+
+  # Separate development and examples from testing requirements
+  patches = [ ./requirements.patch ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    furl
+    jsonschema
+    mock
+    requests
+    xmltodict
```

Comment:
This list doesn't add up with requirements.txt. What is true?

Reply:
[`requirements.txt`](https://github.com/h2non/pook/blob/v1.0.1/requirements.txt) contains

```text
jsonschema>=2.5.1
xmltodict>=0.11.0
furl>=0.5.6
mock>=2.0.0 ; python_version < '3.3'
```

but there are is also the list of [supported clients](https://github.com/h2non/pook#supported-http-clients). Thus, `aiohttp`,  `urllib3` (`requests` pulls it in) and `requests` should be present.

Reply:
Shouldn't mock be only required for less than 3.3?

Reply:
@fabaff

Reply:
Yes, it should. And it seems that the minimal supported Python release is 3.5 (as `aiohttp` is used).